AzureAD/Remove-AadAppsForBc.ps1

```diff
@@ -39,6 +39,16 @@ try {
         Install-Package Microsoft.Graph -Force -WarningAction Ignore | Out-Null
     }
 
+    # Check the AccessToken since Microsoft Graph V2 requires a SecureString
+    $graphAccesTokenParameter = (Get-Command Connect-MgGraph).Parameters['AccessToken']
+  
+    if ($graphAccesTokenParameter.ParameterType -eq [securestring]){
+        $useSecureStringForAccessToken = $true
+        if ($accessToken) {
+            $accessToken = ConvertTo-SecureString -String $accessToken -AsPlainText -Force
+        }

```

Comment:
You convert the accesstoken to a securestring here and then again later
I guess lines 47-49 should be removed?

Reply:
You're right. This happens when you refactor the code several times during development.